src/test/java/org/cactoos/scalar/LengthOfTest.java

```diff
@@ -185,4 +186,15 @@ public void lengthOfEmptyIterator() {
             new IsEqual<>(0)
         ).affirm();
     }
+
+    @Test
+    public void lengthOfText() {
+        new Assertion<>(
+            "Must calculate length of empty iterator",
+            new LengthOf(
+                new TextOf("abcd")
+            ).intValue(),
```

Comment:
@andreoss please, add a test to `LengthOf(...).longValue()`, `LengthOf(...).floatValue()`and `LengthOf(...).doubleValue()`

Reply:
@fabriciofx I did so, but kept it in a single test. Perhaps there could be `IsNumber` matcher for that.

Reply:
@andreoss Please, can you add these tests again and open an issue in `cactoos-matchers` suggesting something like this `isNumber`?

Reply:
@fabriciofx Filed. https://github.com/llorllale/cactoos-matchers/issues/207

Reply:
@andreoss thanks!